hbc.py

```diff
@@ -1,6 +1,9 @@
 import sys
 from PyQt5.QtWidgets import *
 import mainWindow
+from PyQt5 import QtCore, QtWidgets, QtGui
+from PyQt5.QtGui import QIcon,QFont,QPixmap,QPalette
```

Comment:
Why are you importing all of this if you only used QPoint? Sorry if it's a lame question, I'm new with PyQt